charts/urlaubsverwaltung/README.md

```diff
@@ -178,7 +140,7 @@ extraEnv: |
       value: true
     - name: SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE
       value: "true"
-    - name: UV_MAIL_APPLICATIONURL
+    - name: UV_MAIL_APPLICATION-URL
```

Comment:
An environment variable with a dash in it? I mean, it's possible, but not really best-practice